scl/cisco/plugin.conf

```diff
@@ -45,6 +45,10 @@ block parser cisco-timestamp-parser(template()) {
         };
 
         if {
+	    # drops msecs and timezone
+            filter { match('^[.*]?([A-Za-z]{3} [0-9 ]\d \d{2}:\d{2}:\d{2} ((AM)|(PM)))' value('1') flags(store-matches)); };
```

Comment:
I think it's worth adding an example at the top of the config.
`Jun 14 11:57:27 PM.685 UTC`

Reply:
good idea, done.